src/Superdesk/ContentApiSdk/API/Request/RequestInterface.php

```diff
@@ -0,0 +1,146 @@
+<?php
```

Comment:
We could, but imo that would be overkill for what we need. This class represents a API Request, not a generic HTTP request. I might rename it the files name, but i think the namespace already indicates this.